internal/contentcache/contentcache.go

```diff
@@ -44,15 +48,39 @@ func (c *ContentCache) RecoverCache() error {
 	if c.tempDir == "" {
 		c.tempDir = "/tmp"
 	}
+	fmt.Printf("Recovering cache:\n")
```

Comment:
Let's use `logger.Infof` instead of `fmt.Printf` (in all places) because the latter does not write to log files.